router/broker.go

```diff
@@ -88,6 +116,13 @@ func newBroker(logger stdlog.StdLog, strictURI, allowDisclose, debug bool, publi
 		debug:         debug,
 		filterFactory: publishFilter,
 	}
+	err := b.PreInitEventHistoryTopics(evntCfgs)
+	// if broker fails initialize event history store we just log it,
+	// the broker itself will continue to operate but without event history store
+	if err != nil {
+		logger.Printf("error parsing/initializing event history config %v", err)
+		logger.Printf("Event history store won't be working")
```

Comment:
Would it make sense to return the error and allow newBroker to fail?

Reply:
Hm... I agree, returning an error up the stack is right. So I've updated/refactored a bit calling side. Thnx!